var/spack/repos/builtin/packages/chapel/package.py

```diff
@@ -72,8 +72,8 @@ class Chapel(AutotoolsPackage, CudaPackage, ROCmPackage):
 
     patch("fix_spack_cc_wrapper_in_cray_prgenv.patch", when="@2.0.0:")
     patch("fix_chpl_shared_lib_path.patch", when="@2.1:2.2 +python-bindings")
```

Comment:
Have you tested the updated patch against `chapel@2.1`?

Reply:
the `python-bindings` variant isn't actually available until 2.2, so maybe this can/should be revised to `@2.1.1:2.2`?

Reply:
> maybe this can/should be revised to `@2.1.1:2.2`?

Sounds reasonable to me. 

I was mostly worried about validating the patch applies cleanly.